forge-gui/res/cardsfolder/upcoming/zimone_and_dina.txt

```diff
@@ -0,0 +1,11 @@
+Name:Zimone and Dina
+ManaCost:B G U
+Types:Legendary Creature Human Dryad
+PT:3/4
+T:Mode$ Drawn | ValidCard$ Card.YouCtrl | Number$ 2 | TriggerZones$ Battlefield | Execute$ LoseGain | TriggerDescription$ Whenever you draw your second card each turn, target opponent loses 2 life and you gain 2 life.
+SVar:LoseGain:DB$ LoseLife | ValidTgts$ Opponent | LifeAmount$ 2 | SubAbility$ DBGain
+SVar:DBGain:DB$ GainLife | Defined$ You | LifeAmount$ 2
+A:AB$ Repeat | Cost$ T Sac<1/Creature.Other/another creature> | RepeatSubAbility$ DBDraw | MaxRepeat$ 2 | RepeatPresent$ Land.YouCtrl | RepeatCompare$ GE8 | StackDescription$ SpellDescription | SpellDescription$ Draw a card. You may put a land card from your hand onto the battlefield tapped. If you control eight or more lands, repeat this process.
```

Comment:
Why MaxRepeat?? The Oracle doesn't say that

Reply:
Ah, a misleading param name. Does it stop repeating before you run out of (or stop putting) land cards?

Reply:
If only the oracle actually was that clear. I guess if it was a more literal loop you would deck yourself though.

Reply:
I just realized the `SpellDescription$` and oracle text were missing the word "once" in the script! Fixed, sorry about that